src/test/java/com/box/sdk/BoxAPIConnectionTest.java

```diff
@@ -582,17 +579,56 @@ public void getLowerScopedTokenRefreshesTheTokenIfNeededbyCallingGetAccessToken(
 
     @Test
     @Category(UnitTest.class)
-    public void getLowerScopedTokenWithNullResource() {
+    public void getLowerScopedTokenForAPIEndpointResource() {
```

Comment:
Actually, you can combine the following methods into one method, if you parameterize the `resource` string:
- getLowerScopedTokenForAPIEndpointResource
- getLowerScopedTokenForSharedLinkResource
- getLowerScopedTokenForNoteSharedLinkResource